Train.json

```diff
@@ -1079,23 +1066,54 @@
 			"drive": 1,
 			"reliability": 0.95,
 			"cost": 294000,
-			"operationCosts": 120
+			"operationCosts": 70
 		},
 		{
-			"id": 2159,
+			"id": 303,
 			"group": 0,
-			"name": "Stadler EuroDual",
-			"shortcut": "BR 159",
-			"speed": 160,
-			"weight": 123,
+			"name": "DSB EG",
+			"speed": 140,
+			"weight": 132,
+			"force": 400,
+			"length": 21,
+			"drive": 1,
+			"reliability": 0.95,
+			"cost": 450000,
+			"operationCosts": 90
```

Comment:
Hier fehlt ein Komma

Reply:
Schon gesehen, bin schon dran. 
Wie kann ich das denn Uploaden ohne eine neue PR zu erstellen, gibt es die Möglichkeit, das an diesen einfach als Fix anzuheften oder muss ich immer eine neue erstellen?

Reply:
Du kannst einfach neue Commits in die Branche die du hier angeboten hast pushen.